### PR TITLE
Add ErrorLoading test

### PR DESCRIPTION
**Pull request by typeWolffo**

### CODE REVIEW

File: team-fingerprints-web/src/components/ErrorLoading/__test__/ErrorLoading.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  it("should go back after button click", async () => {
+    const history = createMemoryHistory();
+    history.push("/test");
+    renderComponent({ history });
+
+    const button = await screen.findByText("Go back");
+    await userEvent.click(button);
+
+    expect(window.location.pathname).toEqual("/");
+  });
```
Comment (shirve) [inline team-fingerprints-web/src/components/ErrorLoading/__test__/ErrorLoading.spec.tsx:38]: `window.location.pathname` is equal to `"/"` this whole test instead use `history.location.pathname`

File: team-fingerprints-web/src/components/ErrorLoading/__test__/__snapshots__/ErrorLoading.spec.tsx.snap [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
```
Comment (tobob) [inline team-fingerprints-web/src/components/ErrorLoading/__test__/__snapshots__/ErrorLoading.spec.tsx.snap:2]: please don't add snapshots, we don't want them inside project, rather check implementation

**Review by tobob — CHANGES_REQUESTED** [review]
we don't snapshot tests, I don't see any value in them

File: team-fingerprints-web/src/components/ErrorLoading/__test__/ErrorLoading.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  return render(
+    <AppSetup history={history}>
+      <ErrorLoading />
+    </AppSetup>
+  );
+};
+
+describe("ErrorLoading", () => {
+  it("should render properly", () => {
+    render(<ErrorLoading />, { wrapper: BrowserRouter });
```
Comment (DawidHaratyk) [inline team-fingerprints-web/src/components/ErrorLoading/__test__/ErrorLoading.spec.tsx:22]: Wouldn't it be better if you replace it with

` renderComponent();`

as AppSetup component is already wrapped with Router :)

**Review by tobob — APPROVED** [review]

**Pull request merged**